collection/rancher/v2.x/windows-log-collector/README.md

```diff
@@ -0,0 +1,60 @@
+# rancher-windows-log-collector
+Windows log collector for Rancher Windows Worker Nodes
+
+
+## How To Run the Script
+
+```ps1
+Set-ExecutionPolicy Bypass
+Start-BitsTransfer https://raw.githubusercontent.com/rosskirkpat/rancher-windows-log-collector/master/log-collect-beta.ps1
```

Comment:
Need to fix this URL to point to support-tools repo

Reply:
updated the URL to point to the `windows-log-collect` branch in the `support-tools` repo